.github/workflows/sync-github-push-gha-metrics-source.yml

```diff
@@ -40,4 +40,44 @@ jobs:
         with:
           title: Update push-gha-metrics-action
           branch: chore/update-push-gha-metrics-action
-          commit-message: Update push-gha-metrics-action
\ No newline at end of file
+          commit-message: Update push-gha-metrics-action
+
+  update-snapshot:
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+
+    name: Update snapshot version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Collect Metrics
+        id: collect-gha-metrics
+        uses: smartcontractkit/push-gha-metrics-action@v2
+        with:
+          basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
+          hostname: ${{ secrets.GRAFANA_CLOUD_HOST }}
+          this-job-name: Update Version
+        continue-on-error: true
+
+      - name: Checkout the repo
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+        with:
+          ref: snapshot
+
+      - name: Pull changes
+        env:
+          GH_TOKEN: ${{ secrets.GHA_ACTIONS_SOURCE_PULL_TOKEN }}
+        run: ./sync-from-source/sync.sh
```

Comment:
```suggestion
        run: ./sync-from-source/sync.sh --snapshot
```

Reply:
Ah whoops. Thank you.

Reply:
2e20f05850753ea725b6e3ad9a9c70f40fdf4c49